e2e/integration/wordpress-01/__snapshots__/head-tags-wpseo.spec.ts.snap

```diff
@@ -1,92 +1,92 @@
-exports[`Head Tags - WP SEO 0.13 > Post > should render the correct canonical link #0`] = `
+exports[`Head Tags - WP SEO > Post > should render the correct canonical link #0`] = `
```

Comment:
For some reason, these snapshots were not updated and had the wrong name. Someone forgot to merge them before? :shrug: